src/client/components/localisation/view_model.ts

```diff
@@ -52,6 +55,13 @@ export class LocalisationViewModel {
   }
 
   @computed
+  private get balls(): Object3D[] {
+    return this.model.robots
+      .filter(robotModel => robotModel.visible)
```

Comment:
🚗 🔫  If we add a new `@computed` method:

```ts
@computed
private get visibleRobots(): Object3D[] {
     return this.model.robots.filter(robotModel => robotModel.visible)
}
```

Then this can be:
```ts
@computed
private get balls(): Object3D[] {
  this.visibleRobots.map(robotModel => RobotViewModel.of(robotModel).ball)
}
```

and `robots()` could use it too.